src/store/actions/app.ts

```diff
@@ -14,3 +15,12 @@ export type AppStatusToggleActionType = (payload: RequestStatusType) => {
   payload: RequestStatusType;
   type: AppActions.APP_SET_STATUS;
 };
+
+export const toggleAppError: AppErrorToggleActionType = (payload: boolean) => ({
+  type: AppActions.APP_SET_ERROR,
+  payload,
+});
+export type AppErrorToggleActionType = (payload: boolean) => {
+  payload: boolean;
+  type: AppActions.APP_SET_ERROR;
+};
```

Comment:
you can remove `payload: boolean` type from this string. Also as first field in object we use `type` and only then - `payload`